SPECS/kernel/kernel.spec

```
@@ -366,6 +366,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
+* Mon May 23 2022 Neha Agarwal <nehaagarwal@microsoft.com> - 5.15.37.1-3
+- Fix configs to bring down initrd boot time
```

nit - could be more descriptive on which configs are changing
```suggestion
- Remove boot time and mmio tracing configs to bring down boot time
```